platform/osx/os_osx.mm

```diff
@@ -1765,7 +1765,8 @@ static void keyboardLayoutChanged(CFNotificationCenterRef center, void *observer
 OS_OSX* OS_OSX::singleton=NULL;
 
 OS_OSX::OS_OSX() {
-
+	
+	mouse_mode=OS::MOUSE_MODE_VISIBLE
```

Comment:
Well, approved by two experienced Godot devs and merged by a third one, but obviously I should have waited on Travis: it lacks a semicolon, that doesn't compile. :)

Will fix.